src/components/Menu/MenuItem/MenuItem.scss

```diff
@@ -0,0 +1,267 @@
+    height: 36px;
+    width: 36px;
+    position: relative;
+    transition: background-color 0.25s ease-in-out, left 2s;
+  }
+  .menu-item:hover {
+    @media (prefers-color-scheme: light) {
+      .menu-item__icon {
+        color: #FFFFFF;
+        background-color: var(--primaryColor);
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      .menu-item__icon {
+        color: #FFFFFF;
+        background-color: var(--secondaryColor);
+      }
+    }
+  }
+  .menu-item:hover > .menu-item__tooltip {
+    width: 230px;
+  }
+  .menu-item:hover > .menu-item__icon {
+    animation: none;
+  }
+  .menu-item--left:hover > .menu-item__tooltip > .tooltip__text {
+    margin-right: 48px;
+  }
+  .menu-item--right:hover > .menu-item__tooltip > .tooltip__text {
+    margin-left: 48px;
+  }
+  .menu-item--active:hover > .menu-item__icon--end {
+    display: inline;
+  }
+  .menu-item--active:hover > .menu-item__icon--start {
+    display: none;
+  }
+
+  .menu-item__icon {
+    top: 0;
+    left: 0;
+    border-radius: 100%;
+  }
+  @media (prefers-color-scheme: light) {
+    .menu-item__icon {
+      color: var(--primaryColor);
+      background: #FFFFFF;
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    .menu-item__icon {
+      color: #FFFFFF;
+      background: var(--primaryColor);
+    }
+  }
+  
+  .menu-item__icon--end {
+    display: none;
+  }
+
+  .menu-item__tooltip {
+    position: absolute;
+    animation: none;
+    border-radius: 16px;
+    box-sizing: border-box;
+    display: inline-flex;
+    align-items: center;
+    font-family: 'Raleway', serif;
+    font-size: 14px;
+    font-weight: bold;
+    height: 36px;
+    letter-spacing: 1px;
+    line-height: 16px;
+    padding: 0;
+    text-align: right;
+    white-space: nowrap;
+    width: 0;
+    z-index: -1;
+    overflow: hidden;
+    right: 0;
+    transition: width 0.25s ease-in-out;
+  }
+  @media (prefers-color-scheme: light) {
+    .menu-item__tooltip {
+      box-shadow: 0 16px 32px 0 rgba(0, 87, 255, 0.24);
+      background-color: #FFFFFF;
+      color: var(--primaryColor);
+    }
+
+  }
+  @media (prefers-color-scheme: dark) {
+    .menu-item__tooltip {
+      box-shadow: 0 6px 9px 0 rgba(0, 0, 0, 0.2);
+      background-color: var(--primaryColor);
+      color: #FFFFFF;
+    }
+  }
+
+  .menu-item--right > .menu-item__tooltip > .tooltip__text {
+    margin-left: 36px;
+    transition: margin-left 0.5s ease-in-out;
+  }
+  .menu-item--left > .menu-item__tooltip > .tooltip__text {
+    margin-right: 36px;
+    transition: margin-right 0.5s ease-in-out;
+  }
+  @media (prefers-color-scheme: light) {
+    .tooltip__text {
+      color: var(--primaryColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    .tooltip__text {
+      color: #FFFFFF;
+    }
+  }
+
+  .menu-item--left {
+    .menu-item__tooltip {
+      direction: rtl;
+      right: 0;
+      top: 0;
+    }
+  }
+  .menu-item--right {
+    .menu-item__tooltip {
+      direction: ltr;
```

Comment:
I guess that you did not intend to set the read direction `left-to-right` here and just wanted to position the element accordingly on the left or right side of the screen

Reply:
Changed/Removed it ✅